core/templates/dev/head/pages/preferences-page/preferences-page.directive.html

```diff
@@ -214,6 +214,20 @@
       </div>
     </div>
 
+    <hr ng-if="$ctrl.userCanDeleteAccount">
+
+    <div ng-if="$ctrl.userCanDeleteAccount" role="form" class="form-group row">
+      <label class="col-lg-2 col-md-2 col-sm-2">Delete Account</label>
+      <div class="col-lg-10 col-md-10 col-sm-10">
+        <a href="/remove-account">
+          <button type="button" class="btn btn-danger" title="Delete account">
+            Delete account
```

Comment:
Wouldn't this need to be translatable (using i18n strings?)

Reply:
I have created #8088, since this is not yet public I will merge this PR and add the translatability later.